SR2021/2021-01-01-january-update.md

```diff
@@ -0,0 +1,48 @@
+---
+to: SR2021 teams
+subject: Student Robotics January Update
+---
+
+Hello there!
+
+There is less than 2 weeks until our [first round of league matches](https://studentrobotics.org/events/sr2021/league-1/) so we wanted to check in and give you an update.
+
+## League Sessions
+
+The first league session will be happening on the 16th of January, at 12:00 UK time, lasting around 1.5 hours. You will need to have submitted your code to us (more on that below) by 09:00 UK time on the same day. If you do not submit code, or submit late, you will be marked absent and score 0 points.
```

Comment:
Only allowing 3hrs between code submission and live-stream seems quite short. How long did we allow last year?

Reply:
I think we allowed 5 hours -- 10am to 3pm. In practise I think a good portion of that we didn't use though. This year I'd hope we're able to work faster too, given the amount of automation. I agree we should perhaps allow a bit more time for the first of the sessions, in case we need it?

Reply:
Automation should help us massively, but I agree more time for the first is definitely a good idea. Have dialed back to 08:00.

Reply:
:+1: to getting us a bit more time. Not sure if this is a silly question -- does 8am really help? Are we going to be awake at that time in the morning?